cvat/apps/events/permissions.py

```diff
@@ -10,10 +10,12 @@
 from cvat.apps.iam.permissions import OpenPolicyAgentPermission, StrEnum
 from cvat.utils.http import make_requests_session
 
+
 class EventsPermission(OpenPolicyAgentPermission):
     class Scopes(StrEnum):
         SEND_EVENTS = 'send:events'
         DUMP_EVENTS = 'dump:events'
+        SEND_EXTERNAL_EVENTS = 'send-external:events'
```

Comment:
Please update the rego rules to reflect this change

Reply:
I planned it to be only accessible by admins, and rego rules do not need to be changed for it.
But due to comments above I will try to use the existing endpoint somehow, and just drop this scope

Reply:
removed it